### PR TITLE
MAGECLOUD-2899: Redis Slave Nodes Have no Health Checks / Are not Stable

### PR DESCRIPTION
**Pull request by arhiopterecs**
ece-tools version 2002.0.18 and more, uses more fault tolerant settings. In the case when the Magento 2 cannot read data from the Redis slave instance, the reading will be made from the Redis master instance.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@arhiopterecs thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4228) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4228) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by hguthrie** [comment]
Great! @arhiopterecs if you can just rebase to the release branch: 
https://github.com/magento/devdocs/tree/Cloud-2002.0.18

File: guides/v2.1/cloud/env/variables-deploy.md [hunk 1]
```diff
@@ -207,6 +207,8 @@ stage:
 
 You must have a Redis service configured in the `.magento.app.yaml` file and in the `services.yaml` file.
 
+[ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) and more , uses more fault tolerant settings. In the case when the Magento 2 cannot read data from the Redis slave instance, the reading will be made from the Redis master instance.
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/variables-deploy.md]:
```suggestion
[ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) uses more fault-tolerant settings. If Magento 2 cannot read data from the Redis _slave_ instance, then it reads data from the Redis _master_ instance.
```

File: guides/v2.2/cloud/env/variables-deploy.md [hunk 2]
```diff
@@ -224,6 +224,8 @@ stage:
 
 You must have a Redis service configured in the `.magento.app.yaml` file and in the `services.yaml` file.
 
+[ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) and more , uses more fault tolerant settings. In the case when the Magento 2 cannot read data from the Redis slave instance, the reading will be made from the Redis master instance.
```
Comment (hguthrie) [inline guides/v2.2/cloud/env/variables-deploy.md]:
```suggestion
[ece-tools version 2002.0.18]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html#v2002018) uses more fault-tolerant settings. If Magento 2 cannot read data from the Redis _slave_ instance, then it reads data from the Redis _master_ instance.  
```

**Comment by arhiopterecs** [comment]
@hguthrie I changed the base branch.

**Comment by hguthrie** [comment]
hmm...that is scary. Looks like too many files. I won't be able to merge this to the Cloud-2002.0.18 branch because it has master commits...

**Comment by arhiopterecs** [comment]
https://github.com/magento/devdocs/pull/4312 - PR was recreated

**Pull request closed without merging**